Sources/Classes/core/RealMiniApp.swift

```diff
@@ -33,13 +33,14 @@ internal class RealMiniApp {
         miniAppPermissionStorage = MiniAppPermissionsStorage()
         miniAppManifestStorage = MAManifestStorage()
         miniAppDownloader = MiniAppDownloader(apiClient: miniAppClient, manifestDownloader: manifestDownloader, status: miniAppStatus)
-        displayer = Displayer(navigationSettings)
+        displayer = Displayer(settings, navigationSettings)
```

Comment:
Can we rename this to `config` instead of `settings` ? And also the places where we have used. Because so far in SDK, we have shared this object in different classes with `config` naming

Reply:
@rleojoseph if it's ok I'll rename it for displayer to `sdkConfig` because we already had a parameter for the navigationConfig called `config`. so I'd change it to:

```
    init(_ sdkConfig: MiniAppSdkConfig? = nil, _ navConfig: MiniAppNavigationConfig? = nil) {
        self.settings = sdkConfig
        self.navConfig = navConfig
    }
```

Reply:
renamed it as well for `RealMiniApp` but kept it for `MiniApp` so we don't have to rename all `.shared` calls if that's ok